AppKit/CPTextView/CPLayoutManager.j

```diff
@@ -1094,6 +1094,11 @@ var _objectsInRange = function(aList, aRange)
 #pragma mark Init methods
 
 - (id)createDOMElementWithText:(CPString)aString andFont:(CPFont)aFont andColor:(CPColor)aColor
+{
+    return [self createDOMElementWithText:aString andFont:aFont andColor:aColor andUnderline:nil];
+}
+
+- (id)createDOMElementWithText:(CPString)aString andFont:(CPFont)aFont andColor:(CPColor)aColor andUnderline:(CPNumber)aUnderline
```

Comment:
Should you not use `CPUnderlineStyle` instead of CPNumber for the parameter `aUnderline` type?

Reply:
jes, definitely